change/@fluentui-react-button-47eb64e8-6bc3-422a-a77b-c9b271c6a493.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
```suggestion
  "type": "patch",
```

ditto, there is an internal change, but this is not visible to the user

Reply:
It actually affects the API, as ref goes from `React.Ref<HTMLButtonElement | HTMLAnchorElement>` to `React.Ref<HTMLButtonElement | HTMLAnchorElement | HTMLDivElement>`, shouldn't that bump the minor @ling1726?

Reply:
yeah I forgot that you're allowing divs